### PR TITLE
{data,devel}[intel/2017a] intltool v0.51.0, XML-Parser v2.44_01

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/37ff9a4e3e9872f4e68e56702fcfb3e2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2572.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/db9b70c87e566046a4c48e2b83193f61 for a full test report.

**Comment by boegel** [comment]
@easybuilders/easybuild-easyconfigs-maintainers please review

File: easybuild/easyconfigs/i/intltool/intltool-0.51.0-intel-2017a-Perl-5.24.1.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ many different file formats using GNU gettext-compatible PO files."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://launchpad.net/intltool/trunk/%(version)s/+download/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd']
+
+dependencies = [
+    ('XML-Parser', '2.44_01', versionsuffix),
```
Comment (verdurin) [inline easybuild/easyconfigs/i/intltool/intltool-0.51.0-intel-2017a-Perl-5.24.1.eb]: @boegel why no explicit dependency on `Perl` here too?

File: easybuild/easyconfigs/x/XML-Parser/XML-Parser-2.44_01-intel-2017a-Perl-5.24.1.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+]
+
+options = {'modulename': 'XML::Parser'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/x86_64-linux-thread-multi/XML'],
+}
+
+moduleclass = 'data'
```
Comment (verdurin) [inline easybuild/easyconfigs/x/XML-Parser/XML-Parser-2.44_01-intel-2017a-Perl-5.24.1.eb]: @boegel might `tools` be a better `moduleclass`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2048.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1c86b760c708580e6563cdd8b1cc2c39 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2656.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/83c71bac4e4c3237ba340e96da21b463 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/02df70f42a83d3c62a1b811e4539c441 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**